Doc/using/venv-create.inc

```diff
@@ -1,10 +1,12 @@
-Creation of :ref:`virtual environments <venv-def>` is done by executing the
-command ``venv``::
+:ref:`Virtual environments <venv-def>` are created by executing the ``venv``
+command:
```

Comment:
Isn't the _command_ `python`, not `venv`? Venv is a Python module, not a command itself, so calling it a command might lead one to assume that `venv` would work on its own. Seems more accurate to just say:

```suggestion
module:
```